USF-1670: Dropins Localization (#95)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/checkout-dictionary.mdx

---

> title: Checkout Dictionary

Reviewer: ```suggestion
title: Checkout dictionary
```

> description: Reference the default dictionary for the Checkout dropin.

Reviewer: ```suggestion
description: Learn how to customize the default dictionary for the checkout drop-in component.
```

---

import CodeInclude from '@components/CodeInclude.astro';
import CodeImport from '@components/CodeImport.astro';
import dictionary from '@dropins/storefront-checkout/i18n/en_US.json.d.ts?raw';

> The default Checkout dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the Checkout drop-in.

Reviewer: ```suggestion
The default checkout dictionary file (`i18n/en_US.json`) contains the default values for all labels and text common to the checkout drop-in component.
```
